### DISCUSSION TITLE
[Request] Kits paging + Group kits cooldown

### DISCUSSION THREAD

**Discussion by wingcwf** [Zrips/CMI#644] [Feature Requests]
Too many Kits, it started to get messy

I hope to do the paging function
Example:
![kit](https://user-images.githubusercontent.com/34314256/43072853-1a16cd7a-8eaa-11e8-9e49-ad7cbbd28b40.jpg)

![kit](https://user-images.githubusercontent.com/34314256/43072917-48b2b90a-8eaa-11e8-8bd6-39a8cae218ed.jpg)

![kit](https://user-images.githubusercontent.com/34314256/43072878-2c6b543c-8eaa-11e8-8061-0731b841daad.jpg)

![kit](https://user-images.githubusercontent.com/34314256/43072901-38238df8-8eaa-11e8-83d2-defc592e7800.jpg)

-------------------------------------------------------------------------------------------------------------------
Different permission groups have different cooldown times
Like default (7d) vip (5d) etc.

**Comment by CloudeLecaw** [comment]
i see u  wanted subgroups of kits, a nice idea :-P

currently Kits are made with permission the layer has…


When the Suggestion is nosense:
Currently if u have 60 Kits and the Player just has permission for 10 then  he just see the 10 kits!
it is not nessesary to Show him Donator Kits in the Gui of he has no Access for it....or is it?....
Your Rank and Permissionsystem should mange the Access for the sperator Ranks, Free, VIP,what everGroup
Never test this as Admin: why?...hm u see all Kits as Admin or OP.. so give urself the desired rank for testing what Kits u have Access for here..
 
When the Suggestion is may usefull:...
If u have 60 Kits and the Player sees all 60 Kits percause he has permission for it.
uhhhh….. then this feature u suggest would be a nice welcome ;-)
.
.
Or u just wanne hypocrite the FreePlayers how much cool stuff they get if they pay the Donatorrank?...
well this is what a System like this would definatly do ;)...hypocrite "Ohh u get 299 Kits if u become Donator, look here what cool thinks you get, ohh your just a FreePlayer? then u just get these specia five grasblock kits"
. 
.
..... lol..sorry…i went a bit ooT just was writing what i think of payedDonatorRank, yeah i know… some owners dont have enough Money and Need this to further exists, so it is ok.. :p
mostly they say "i dont need it, but why not take some extra Money for all my work, blabla…."
The only one that really does a lot of work here from all of us is Zrips and his ModGroup here on GitHub ;) They have a lot of work with reading our strange ideas, comments and think about them (or may not ^^)

**Comment by wingcwf** [comment]
I have not said want subgroups of kits = =...
I just want have paginate , The picture above is just a screenshot on other server..I am just using it for example

**Comment by CloudeLecaw** [comment]
You should post other screenshots then if u dont want a Grouping and Subgrouping
Your Screenshots Show Subgroups no Paging or other function; (and they are tin the middle so we could speak of some kind like  design how kits are displayed in the chestgui)

Paging would be a listing of all Kits and in the last line two Buttons_ _Next Page/Prev Page_

Subgrousp what your Screenshot show's:
Group1:   Free ->  Kit1, Kit2, Kit3, Kit4- Kit5
Group2:   Donator  -> Kit6, Kit7, Kit8, Kit9- Kit10

Belive it or not but in the backgroup you have still this 10 Kits in your files and 1,2,3,4,5 are a Sub of Free and 6,7,8,9,10 a Sub of Donator.
